go/vt/servenv/exporter.go

```diff
@@ -307,6 +307,10 @@ func (e *Exporter) NewGauge(name string, help string) *stats.Gauge {
 	return lvar
 }
 
+func (e *Exporter) NewGaugeFloat64(name string, help string) *stats.GaugeFloat64 {
+	return nil
```

Comment:
I don't follow why this is returning `nil`. Maybe add a comment?

Reply:
See original comment, where this is explained. I'll also add as code comment.